.github/workflows/build.yml

```
@@ -0,0 +1,36 @@
+name: Build
+on: [push]
```

```suggestion
on:
    pull_request:
    push:
        branches:
            - streamflow
            - next
```

^ So we can run CI off any PR including those from forks while continuing to run for pushes to the `streamflow` and `next` branches. This would be similar to how CI works for [go-livepeer](https://github.com/livepeer/go-livepeer/blob/849c8bf43a73ce8dcc530f044cb8642192ff1331/.github/workflows/linux.yml#L3).